pandas/tests/arrays/test_period.py

```diff
@@ -88,24 +87,29 @@ def test_take_raises():
         arr.take([0, -1], allow_fill=True, fill_value='foo')
 
 
-@pytest.mark.parametrize('dtype', [int, np.int32, np.int64])
+@pytest.mark.parametrize('dtype', [int, np.int32, np.int64, 'uint'])
```

Comment:
The addition of uint will cause this to fail. Removing, since it's tested elsewhere now.

Though those tests are just index. I'll add ones for arrays as well...

Reply:
Fixed in 5fa32e9848406fc2f9f5a1608a7e3545df70b06c. The test is slightly more complicated now.